std-bits/table/src/main/java/org/enso/table/problems/AggregatedProblems.java

```diff
@@ -53,6 +53,16 @@ public void add(Problem problem) {
     count++;
   }
 
+  public void addAll(List<Problem> problems) {
+    for (Problem p : problems) {
+      add(p);
+    }
+  }
+
+  public void addAll(AggregatedProblems that) {
+    addAll(that.problems);
+  }
```

Comment:
Nice and I guess useful, but is that actually used anywhere? I could not find.

Reply:
Removed; they were used earlier.